tasks/deploy/lib/constants.ts

```diff
@@ -3,6 +3,16 @@ import { Address } from "viem";
 export const ETH_ADDRESS =
     "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE" as Address;
 
+// ╭─────────────────────────────────────────────────────────╮
+// │ Linea Addresses                                       │
```

Comment:
need some more spaces after Addresses (I also need to add blurb on commenting plugin since I forgot to do that)